ledger/metrics/src/main/scala/com/daml/metrics/HttpJsonApiMetrics.scala

```diff
@@ -25,43 +30,78 @@ class HttpJsonApiMetrics(val prefix: MetricName, override val registry: MetricRe
 
   val surrogateTemplateIdCache = new CacheMetrics(prefix :+ "surrogate_tpid_cache", registry)
 
+  val dropWizardFactory = new DropwizardFactory {
```

Comment:
Nit: To be consistent with the spelling of the underlying metrics library (i.e., `Dropwizard` or `dropwizard`), I'd suggest spelling this similarly as `dropwizardFactory`.